CMakeLists.txt

```diff
@@ -130,10 +130,12 @@ if(CMAKE_COMPILER_IS_MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /fp:precise /wd4800 /wd4521 /wd4251 /wd4275 /wd4305 /wd4355 ${SSE_FLAGS_STR}")
 
     # Add extra code generation/link optimizations
-    if(CMAKE_MSVC_CODE_LINK_OPTIMIZATION)
+    if(CMAKE_MSVC_CODE_LINK_OPTIMIZATION AND (NOT BUILD_CUDA) AND (NOT BUILD_GPU))
```

Comment:
Something feels wrong in the diff. It still shows the old version as the original, but shows updated version in the incoming changes

Reply:
Sorry, I had trouble with merging. I reverted the branch.